Bundle/WidgetBundle/Resolver/WidgetResolver.php

```diff
@@ -77,7 +80,10 @@ protected function assert($value, $operator, $expected)
         $result = false;
         switch ($operator) {
             case self::OPERAND_EQUAL:
-                $result = $value === $expected;
+                $result = $value == $expected;
```

Comment:
It would be safer to convert $expected to true or false when equals to 1 or 0, then do a === comparison.
== comparison seems not sufficient strict because of true == 'false' returning true